### DISCUSSION TITLE
Habe mein Passwort für Stronghold Backup gelöscht

### DISCUSSION THREAD

**Discussion by MrAzorm** [iotaledger/firefly#7441] [Help & Support]
Hallo zusammen,
ich war so dämlich und habe mein abgelegtes Passwort gelöscht, brauche es aber wieder um in Firefly herein zu kommen, kann man aus alten Stronghold Backup das Passwort herauslesen? Kann ich mir ein neues vergeben? 
Bitte um Hilfe

Vielen Dank

**Comment by begonaalvarezd** [comment]
Hi, if you lost your password you cant recover it, but you can always have access to your funds using your 24 words mnemonic. Just create a new profile using your mnemonic and set a new password for it.
